lib/controller.js

```diff
@@ -616,6 +616,20 @@ controller.getWifiInfo = function(opts) {
   });
 };
 
+controller.getMemoryInfo = function(opts) {
+  opts.authorized = true;
+  return controller.standardTesselCommand(opts, function(tessel) {
+    return tessel.memoryInfo()
+      .then(function(meminfo) {
+        Object.keys(meminfo).forEach(function(key) {
+          logs.basic(sprintf('\t%s\t%d kB', key, meminfo[key] / 1000));
+        });
+
+        return controller.closeTesselConnections([tessel]);
```

Comment:
Well, this is unnecessary anyway, since we're not exposing this as a user facing command now :)